README.md

```diff
@@ -232,7 +232,19 @@ value => {
     return item => true;
 }
 ```
+#### Right column Search
```

Comment:
Please separate PRS.
This PR is to lock items in the selected items list? so why add the search feature here?
Also, I'm not seeing this filterRightSearch (change name) in the readme.